src/main/java/edu/illinois/cs/dt/tools/minimizer/TestMinimizer.java

```diff
@@ -99,11 +104,14 @@ public MinimizeTestsResult run() throws Exception {
 
             if (ONE_BY_ONE_POLLUTERS) {
                 info("Getting all polluters (dt.minimizer.polluters.one_by_one is set to true)");
-                for (List<String> order : getSingleTests(fullTestOrder, dependentTest)) {
+                List<String> testOrderToGetPolluters = fullTestOrder;
+                if (CUSTOM_POLLUTERS != "") testOrderToGetPolluters = new ArrayList<String>(Arrays.asList(CUSTOM_POLLUTERS.split(";")));
```

Comment:
Please have the if statement be split into multiple lines and include curly brackets, even if it's just one statement. Same for the if statement below.

Reply:
Sure. Should be fine now.